library/src/androidTest/java/com/google/android/exoplayer/MediaFormatTest.java

```diff
@@ -45,20 +45,20 @@ public void testConversionToFrameworkFormat() {
     initData.add(initData2);
 
     testConversionToFrameworkFormatV16(MediaFormat.createVideoFormat(
-        MediaFormat.NO_VALUE, "video/xyz", 5000, 102400, 1000L, 1280, 720, initData));
+            null, MediaFormat.NO_VALUE, "video/xyz", 5000, 102400, 1000L, 1280, 720, initData));
```

Comment:
Please put the indentation back to what it used to be like (4 chars) (ditto x2 below).